gatsby/src/components/DrugSections/IndicationsAndDose/IndicationsAndDose.tsx

```diff
@@ -71,9 +46,26 @@ export const IndicationsAndDose: FC<IndicationsAndDoseProps> = ({
 					key={content.contentFor}
 					content={content}
 					collapsible={collapsible}
-					defaultOpen={defaultOpen}
 				/>
 			))}
+		</>
+	);
+
+	return (
+		<section aria-labelledby={slug} className={styles.wrapper}>
+			<h2 id={slug} dangerouslySetInnerHTML={{ __html: potName }} />
+
+			{collapsible ? (
+				<AccordionGroup
+					toggleText={(isOpen) =>
+						`${isOpen ? "Hide" : "Show"} all ${numberOfSections} sections`
+					}
+				>
+					{bodyContent}
+				</AccordionGroup>
+			) : (
+				bodyContent
```

Comment:
Is this missing brackets?

Reply:
Oh it's just cos it's in a ternary and formatted weirdly I guess cos of prettier over multiple lines